### ISSUE TITLE
Create Calendar Month Markdown Table

### ISSUE THREAD

**Issue by MartinPacker** [MartinPacker/mdpre#15]
Candidate syntax:

```
=cal November 2021
=endcal
```

**Comment by MartinPacker** [comment]
Actually more like `=cal 2021 11`.

And I think we can highlight with a CSS class particular days:

```
=cal 2021 11
=caldays blue 14 15 16 17
=caldays green 18
=endcal
```
`blue` and `green` are classes - and the day  numbers would be wrapped with `<span>` elements with `class=` set.

**Comment by MartinPacker** [comment]
Added syntax:

```
=calnote 17 A
```

adds annotation to calendar entry for 17th day.

This annotation can have a `<span>` in it to colour it.

Also added

```
=calkeys blue Day we have data for
```

which generates a bullet coloured using class `blue` and text `Day we have data for` to act as a kind of key to colouring.

**Comment by MartinPacker** [comment]
A third parameter to `=cal` defines how many lines each row should occupy:

```
=cal 2021 11 3
```

Gives 3 lines in a row. The default is 2.

Here's a complete test calendar:

```
### November 2021
<!-- md2pptx: contentSplit: 2 1 -->

=cal 2021 12 2
=caldays blue 14 15 16 17
=caldays orange 18
=calkey blue We have data for **somehow**
=calkey orange Is focus day
=calnote 17 A
=calnote 18 <span class="green">B</span>
=calnote 4 X
=endcal

* An unrelated bullet
  * An unrelated sub-bullet
```

Obviously - with the md2pptx dynamic `contentSplit` specification - this tests well with md2pptx.

**Comment by MartinPacker** [comment]
Shipped in v0.6.
